roles/network_plugin/calico/templates/calico-typha.yml.j2

```diff
@@ -124,6 +116,16 @@ spec:
             value: "true"
           - name: TYPHA_PROMETHEUSMETRICSPORT
             value: "{{ typha_prometheusmetricsport }}"
+{% endif %}
+        volumeMounts:
```

Comment:
Why don't we put this `volumeMounts` line into `{% if typha_secure %}` section?
The next section is livenessProbe, if not typha_secure this`volumeMounts` line only is there.

Reply:
I see what you mean. I thought that there were more volume mounts present. I will make that change now.